<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/forUI/OPROSNIK_MATER.png);
</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>245</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
background-image: url(:/forUI/back.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>245</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/forUI/back.png);
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>309</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/forUI/back.png);
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>309</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/forUI/back.png);
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>461</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/forUI/back.png);
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>352</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/forUI/back.png);
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>352</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/forUI/back.png);
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_8">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>461</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/forUI/back.png);
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../pppp.qrc"/>
 </resources>
 <connections/>
</ui>
